devchat.go

```diff
@@ -92,7 +95,12 @@ func main() {
 			"If you still can't join, try joining back in 2 minutes. If you _still_ can't join, make an issue at github.com/quackduck/devzat/issues", true)
 		os.Exit(0)
 	}()
-
+	go func() {
```

Comment:
Using this approach (refreshing the entire map each second) can lead to an unexpected side effect:

Suppose someone posts 10 messages in 50 seconds half a second before the map is due to be refreshed, then, even tho they _should_ be banned, they won't since the map gets refreshed mid-way. To fix this, edit the part where the message counter is incremented and make it decrease that ID's count after one second. (use `time.AfterFunc`)